### ISSUE TITLE
Azure Monitor/Dashboard: Panel TimeSeries link does not properly forward to Azure Log Analytics (subscription ID missing in URL)

### ISSUE THREAD

**Issue by kforeverisback** [grafana/grafana#52294]
**What happened**:

Azure Log Analytics link in a TimeSeries table does not properly forward.
When clicked it shows unexpected error in Azure Log Analytics (as shown below).
Upon inspecting the URL generated by Grafana, it seems the subscription ID is absent from the URL.

See the GIF below:
![Grafana_Link_Issue](https://user-images.githubusercontent.com/11367790/179090789-5948e14e-4294-4d8a-85c6-0dd115349d32.gif)

**What you expected to happen**:
Upon clicking Azure links, it should properly forward it to a Sharable Log Analytics Query Link.

**How to reproduce it (as minimally and precisely as possible)**:

Create a dashboard Panel with a Kusto querying from Log Analytics, formatted as a TimeSeries data.
![image](https://user-images.githubusercontent.com/11367790/179092945-5dea4057-8957-4358-a046-a371bea1432b.png)

Sample query:

```kusto
customLog_CL
| where $__timeFilter(Date_t) 
   and container == 'container'
   and k_app_s == 'some-app'
   and log == "log.name" 
   and Zone has_any ('eastus','westus2')

| summarize percentile(Duration_d, 75), percentile(Duration_d, 90), percentile(Duration_d, 95), percentile(Duration_d, 98) by Zone_s, k_app_s
| extend Zone=Zone, 75th=round(percentile_Duration_d_75,2), 90th=round(percentile_Duration_d_90,2), 95th=round(percentile_Duration_d_95,2), 98th=round(percentile_Duration_d_98,2), AppType=k_app_s
| project AppType, Zone, 75th, 90th, 95th, 98th
| order by AppType, Zone asc
```

**Anything else we need to know?**:

Upon inspecting the URL generated by Grafana, the subscription id seems to be missing.

Example URL Generated by Grafana:
```
%7B%22resources%22%3A%5B%7B%22resourceId%22%3A%22%2Fsubscriptions%2F%2Fresourcegroups%2Frg-ngsa-asb-dev%2Fproviders%2Fmicrosoft.operationalinsights%2Fworkspaces%2Fla-aks-avcdefghijkl
                                                                   ^
                                                                   | Subscription ID misssing
```

Modified above URL with subscription ID added:
```
%7B%22resources%22%3A%5B%7B%22resourceId%22%3A%22%2Fsubscriptions%2F698dcb5a-ae1e-b8b2-cf6b-fc6ef28a315f%2Fresourcegroups%2Frg-ngsa-asb-dev%2Fproviders%2Fmicrosoft.operationalinsights%2Fworkspaces%2Fla-aks-avcdefghijkl
                                                                   ^
                                                                   |Here it is added manually
```

**Environment**:
- Grafana version: 8.4.7, 8.5.5
- Data source type & version:
- OS Grafana is installed on: Grafana Docker
- User OS & Browser: Edge, Chrome, Firefox
- Grafana plugins: Azure Monitor
- Others:

**Comment by andresmgot** [comment]
Thanks for the report @kforeverisback, I can confirm this is not working. This URL is generated based on a `subscriptionId` and a `workspace` here:

https://github.com/grafana/grafana/blob/ecaa1dcbfd86d33b816edef2f17d48775fe75ef7/public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts#L195-L200

Which is set in the backend here:

https://github.com/grafana/grafana/blob/b10ddfdf8c4213e944f5d3f902d71df43a82e581/pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource.go#L200-L203

But those parameters are no longer used, we should use the `resourceURI` instead (legacy queries may still use the `workspace` URL).

**Comment by kforeverisback** [comment]
> Thanks for the report @kforeverisback, I can confirm this is not working. This URL is generated based on a `subscriptionId` and a `workspace` here:
> 
> https://github.com/grafana/grafana/blob/ecaa1dcbfd86d33b816edef2f17d48775fe75ef7/public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts#L195-L200
> 
> Which is set in the backend here:
> 
> https://github.com/grafana/grafana/blob/b10ddfdf8c4213e944f5d3f902d71df43a82e581/pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource.go#L200-L203
> 
> But those parameters are no longer used, we should use the `resourceURI` instead (legacy queries may still use the `workspace` URL).

Thank you @andresmgot for looking into this. Much appreciated.

**Comment by Deltids** [comment]
We have the same type of issue, but no links are generated anymore. See image for no underline. 

v9.1.4 (2186d0bbeb) in Docker

![image](https://user-images.githubusercontent.com/20363682/189849582-0cd4ba16-f04c-42fd-94f1-65bc4bd98c5f.png)

**Comment by aangelisc** [comment]
This issue should be fixed and the fix should be released in the next 9.3 patch version 😊

**Comment by kforeverisback** [comment]
@aangelisc W00T! Thanks a ton for this.
And thanks for your contribution!
